lib/handle-pull-request-change.js

```diff
@@ -1,17 +1,29 @@
 module.exports = handlePullRequestChange
 
 async function handlePullRequestChange (robot, context) {
-  const title = context.payload.pull_request.title
-  const isWip = /\b(wip|do not merge)\b/i.test(title)
+  const {title, html_url, head} = context.payload.pull_request
+  const isWip = containsWIP(title) || await commitsContainWIP(context)
   const status = isWip ? 'pending' : 'success'
 
-  console.log(`Updating PR "${title}" (${context.payload.pull_request.html_url}): ${status}`)
+  console.log('Updating PR "%s" (%s): %s', title, html_url, status)
```

Comment:
Any particular reason for switching from `${template literals}` ?
This PR is great btw. :+1: :smile:

Reply:
that was more by accident 😁 At some point during refactoring I had this

```js
console.log('Updating PR "%s" (%s): %s', 
  context.payload.pull_request.title, 
  context.payload.pull_request.html_url, 
  context.payload.pull_request.status)
```

Putting it all in one line made it less readable. Now that it's just `console.log('Updating PR "%s" (%s): %s', title, html_url, status)` we could as well have left it at template literals. Feel free to send a pull request if you like, I think template literals make it more readable :)

Reply:
Sure @gr2m thanks! :tada: 
Sending one. :+1: